code/API_definitions/quality-on-demand.yaml

```diff
@@ -436,12 +436,64 @@ paths:
         "403":
           $ref: "#/components/responses/Generic403"
         "404":
-          $ref: "#/components/responses/SessionNotFound404"
+          $ref: "#/components/responses/Generic404"
         "500":
           $ref: "#/components/responses/Generic500"
         "503":
           $ref: "#/components/responses/Generic503"
 
+  /sessions/retrieve:
```

Comment:
I also raised this point for the QoS provisioning proposal, but the design guidelines could be interpreted as meaning this endpoint must be named `POST /retrieve-sessions`.

Reply:
I also commented there, that both names work for me, so no strong opinion here

Reply:
I have changed it to /retrieve-sessions